WordPress/Classes/ViewRelated/Support/SupportTableViewController.swift

```diff
@@ -196,20 +219,24 @@ private extension SupportTableViewController {
     // MARK: - ImmuTableRow Struct
 
     struct HelpRow: ImmuTableRow {
-        static let cell = ImmuTableCell.class(WPTableViewCellValue1.self)
+        static let cell = ImmuTableCell.class(WPTableViewCellIndicator.self)
 
         let title: String
+        let showIndicator: Bool
         let action: ImmuTableAction?
 
-        init(title: String, action: @escaping ImmuTableAction) {
+        init(title: String, action: @escaping ImmuTableAction, showIndicator: Bool = false) {
             self.title = title
+            self.showIndicator = showIndicator
             self.action = action
         }
 
         func configureCell(_ cell: UITableViewCell) {
+            let cell = cell as! WPTableViewCellIndicator
```

Comment:
I started to ask about `guard`ing the cast but in this case I think the force unwrap is perfectly safe. :)